lib/components/link.vue

```diff
@@ -13,8 +13,7 @@
        :event="event"
        :tag="tag"
        :class="linkClassObject"
-       @click="linkClick"
-    >
+       @click="linkClick">
```

Comment:
This would probably fix quite a few things.  and might allow us to clean out some code in related components (i.e. nav-item`, dropdown-item`, etc) that use `@click.native` to get around this.